### DISCUSSION TITLE
Possible to have single video playback for entire day?

### DISCUSSION THREAD

**Discussion by dlong500** [blakeblackshear/frigate#6428] [Ask A Question]
Is it possible to adjust settings in the "Recordings" UI to have a single video per day instead of it broken up into hourly segments?  For casual viewing I'd like to be able to seek through an entire day at a time.

**Comment by NickM-27** [comment]
As soon as a recording hour ends the next one will begin playing 
 automatically

**Comment by dlong500** [comment]
I do understand that.  It would just be convenient for my 24/7 setup to be able to have a single seek bar for the day.  For example, as it is now, if I wanted to jump ahead a few hours I'd have to either go to the menu and select the proper time segment or seek to the end of the currently playing segment several times.

**Comment by blakeblackshear** [comment]
This will likely be addressed in the 0.14 release.

**Reply by dlong500** [reply]
That's great!  Is there a planned ETA for future releases?  Just curious--no pressure--I know things take time.  This appears to be the only issue or discussion that the 0.14 release is mentioned so far.

**Reply by NickM-27** [reply]
0.13 is still in progress
